src/slicegpt/adapters/sliced_phi.py

```diff
@@ -0,0 +1,56 @@
+import torch
+import torch.nn as nn
+from transformers.models.phi.modeling_phi import PhiConfig, PhiForCausalLM, PhiModel
+
+from slicegpt.adapters.phi2_adapter import CompressedPhiDecoderLayer, Phi2ModelAdapter
+from slicegpt.modules import RMSN
+from slicegpt.rotate import slice_rotated_model
+from slicegpt.slicing_scheduler import SlicingScheduler
+
+
+class SlicedPhi(PhiModel):
```

Comment:
I wonder if we should change the `model_type` to `sliced_phi` along the lines of `quarot_phi` etc, but that requires us to extend the config class. Let's leave this for a future PR.

Reply:
Added the configs and model type